packages/fluttium_cli/lib/src/bundles/fluttium_action_bundle.dart

```diff
@@ -5,22 +5,78 @@ import 'package:mason/mason.dart';
+      "type": "text"
+    },
+    {
+      "path": "{{name.snakeCase()}}/.dart_tool/package_config_subset",
```

Comment:
same here we should probably ignore the .dart_tool directory

Reply:
How do we do that 👀